test/e2e/apimachinery/chunking.go

```diff
@@ -122,6 +122,29 @@ var _ = SIGDescribe("Servers with support for API chunking", func() {
 		gomega.Expect(list.Items).To(gomega.HaveLen(numberOfTotalResources))
 	})
 
+	ginkgo.It("should chunk lists of Pods", func() {
+		ns := f.Namespace.Name
+		c := f.ClientSet
+		client := c.CoreV1().PodTemplates(ns)
```

Comment:
did you mean pods? otherwise the test title is wrong

Reply:
we shouldn't use PodTemplates objects in new tests. see https://github.com/kubernetes/kubernetes/issues/82435